backend/src/tests/user.test.ts

```diff
@@ -85,8 +95,18 @@ describe('user router logout', () => {
         res = await api
             .post('/api/user/logout/')
             .set('Cookie', setCookie)
-            .expect(401);
-        expect(res.headers['set-cookie']).toBe(undefined);
+            .expect(204);
+        setCookie = res.headers['set-cookie'] as string[];
+        expect(typeof setCookie).toBe('object');
+        //check that the cookie is overwritten properly
+        const tokenCookie = setCookie.find((e) => e.startsWith('user-token='));
+        expect(typeof tokenCookie).toBe('string');
+        const splitted = (tokenCookie as string).split('; ');
+        expect(splitted.includes('HttpOnly')).toBe(true);
+        expect(splitted.includes('Secure')).toBe(true);
+        expect(splitted.includes('SameSite=Strict')).toBe(true);
+        const token = splitted[0].slice('user-token='.length);
+        expect(jwt.decode(token)).toBe(null);
```

Comment:
choose whether to uncomment code or remove it

Reply:
The code is currently commented out because these tests require database and can not therefore be run yet. I added these new lines of commented code since the specs changed and these are now the tests we want to run once it is again possible. If the commented parts would be removed or left outdated someone would have to implement them again later.